docs/css/style.css

```diff
@@ -180,15 +180,18 @@ a.direct-link {
 
 ul {
   box-sizing: content-box;
-  column-count: 2;
-  column-gap: 30px;
   margin-top: 20px;
   padding: 0 15px;
```

Comment:
[Refactor] The `.single-column` class below overrides the `margin-top` and `padding` styles too. Is there a reason not to move these specific values into `.two-column` as well?